.github/workflows/org-quality-gate.yml

```
@@ -0,0 +1,43 @@
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      actions: read
+      security-events: write
+    steps:
+      - name: CodeQL handled by reusable gate
+        run: echo "CodeQL is enforced by .github/workflows/code-quality.yml"
+
+  missing_reusable_gate:
+    name: missing_reusable_gate
+    if: ${{ hashFiles('.github/workflows/code-quality.yml') == '' }}
```

actionlint (1.7.11)

[error] calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)